services/twofa.go

```diff
@@ -0,0 +1,283 @@
+package services
+
+import (
+	"bytes"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"image/png"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/brave/accounts/datastore"
+	"github.com/brave/accounts/util"
+	"github.com/google/uuid"
+	"github.com/pquerna/otp"
+	"github.com/pquerna/otp/totp"
+	"github.com/rs/zerolog/log"
+)
+
+const (
+	totpIssuerEnv = "TOTP_ISSUER"
+	totpQRSizeEnv = "TOTP_QR_SIZE"
+	defaultIssuer = "Brave"
```

Comment:
```suggestion
	defaultIssuer = "Brave Account"
```

Reply:
We should test that the `totp` library correctly URL-encodes that to `Brave%20Account`.